cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -38,15 +38,15 @@ export interface Auth {
    *
    * @param redirect - location to redirect after successful authentication
    */
-  authenticate(redirect: string): express.Handler;
+  authenticate(redirect: string, loginError: string): express.Handler;
```

Comment:
Need jsdocs for second param, maybe rename to `errorRedirect`? I had to look for the usage to tell what it was, my initial guess was that it was an error message :joy: